### PR TITLE
Wrap g apps hook

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
###### Motivation for this change

uses the more correct hook instead of NIH-gnome-program-wrapping
###### Things done
- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by bendlas** [comment]
previous discussion #16183

**Comment by lethalman** [comment]
Checking the travis output, `gnome-bluetooth` fails to build. Did you actually test your change?
```
installing
install flags: install    gsettingsschemadir=/nix/store/10s8hrw09y02sq0r9rmlskggn8nfqib2-gnome-bluetooth-3.18.2/share/gsettings-schemas/gnome-bluetooth-3.18.2/glib-2.0/schemas/
make: *** No rule to make target 'install'.  Stop.
builder for ‘/nix/store/brpvjc86wsqwg417a3y0jdrif7v67xr7-gnome-bluetooth-3.18.2.drv’ failed with exit code 2
```

Perhaps it's better to avoid `wrapGAppsHook` in gnome-bluetooth for now.

**Comment by RonnyPfannschmidt** [comment]
It works locally, im not sure when i can resume the experiment

**Comment by lethalman** [comment]
@RonnyPfannschmidt indeed I've tried locally and it works.

**Comment by RonnyPfannschmidt** [comment]
i resumed the check, and the binary is broken it seems :/

**Comment by lethalman** [comment]
@RonnyPfannschmidt you mean it compiles but the wrapper is broken?

**Comment by RonnyPfannschmidt** [comment]
correct - atm i can no longer use bluetooth

**Comment by lethalman** [comment]
@RonnyPfannschmidt ok let's do it step by step, remove the bluetooth patch from this PR so I can merge it

File: pkgs/desktops/gnome-3/3.18/core/gnome-online-accounts/default.nix [hunk 1]
```diff
@@ -1,7 +1,7 @@
 { stdenv, fetchurl, pkgconfig, glib, libxslt, gtk, makeWrapper
```
Comment (SShrike) [inline pkgs/desktops/gnome-3/3.18/core/gnome-online-accounts/default.nix]: The `makeWrapper` import is still here, but it's never used.

**Comment by RonnyPfannschmidt** [comment]
@lethalman i force-pushed to remove the latest commits, please have a look

**Comment by lethalman** [comment]
Built locally, thanks.

**Comment by lethalman** [comment]
Backported to release-16.03 as requested by @RonnyPfannschmidt

**Pull request merged**